### PR TITLE
First draft of D.Ugol portfolio page

### PR DESCRIPTION
**Pull request by d-ugol**
This is the first draft of the web page. Please review.

### CODE REVIEW

**Comment by candre717** [comment]
LGTM

**Comment by PatriciaMuench** [comment]
I suggest putting your name as the title, but it looks pretty good so I don't mind if you merge it like this. P.S. I want to pet an 🦉 .

File: dugol.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<html>
+    <head>
+
+
+      <title>Startup Institute Web Development</title>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+
+<!--
+
```
Comment (chdastolfo) [inline dugol.html]: There's nothing in between these comment lines, so you can remove them.

**Comment by chdastolfo** [comment]
There's some extra whitespace and cruft in dugol.html that you can remove, but other than that this looks good! I'm going to ask you to hold off on merging - as you can see, there's a merge conflict here, so I'm going to have us all resolve these in class on Thursday. Mwahahaha.

**Pull request merged**